front-end/public/index.html

```diff
@@ -28,6 +28,8 @@
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
+    <script src="https://www.paypal.com/sdk/js?client-id=AfhxQwSt-H9J5KyiJGaXvjfYO7-IJDh0rvfuhoAlSwB3pnLvxYTOlnFtFocGbiBUg5gc_tILQSdWMvZH"></script>
```

Comment:
Scripts are loaded after the content at the bottom of the page. In react html is rendered in the div with id root, so script tags should be loaded under it.